README.md

```diff
@@ -1,5 +1,13 @@
 # Safer
 
+Safer is a tool that helps you sign & submit transactions to a Gnosis Safe, without requiring any interaction with Gnosis' backend or frontend.
+
+Safer is a set of Foundry scripts that can be used as fallback in case Gnosis is down and the blockchain is the only thing that can be trusted.
```

Comment:
```suggestion
Safer is a set of Foundry scripts that can be used as fallback in case Safe backend is down and the blockchain is the only thing that can be trusted.
```